### ISSUE TITLE
Issue in training a navigation policy

### ISSUE THREAD

**Issue by sreeharshaparuchur1** [facebookresearch/soundspaces-challenge#10]
Hi @SAGNIKMJR ,

I followed the instructions at the challenge branch of the Active Audio Visual Dynamic Sound Separation GitHub repository to train my own navigation policy using the pretrained audio separator. 

The command I used to train my policy is: 

```Training command: CUDA_VISIBLE_DEVICES=0 python3 main.py --exp-config audio_separation/config/train/nearTarget.yaml --model-dir runs/active_train/near_target --run-type train NUM_PROCESSES 1```

I had to make modifications to the command given in the README due to my hardware limitations. 

Training started off fine, however, it ended after 1341020 frames without an exit error being thrown. The paper says that the policy in AAViDSS was trained for 150 million steps, so I'm confounded as to why my programme ended at a mere 1.3 million steps. Could you tell me why this might be the case and what I can do to ensure my policy is trained correctly?

Also interesting to note was the fact that the policy reward seemed to have stagnated at around 600k frames. This is odd because the train split itself contains 700+k episodes, each with 20 steps, thus, I feel that the loss should have been teetering at that point. It would be helpful if you could upload the tensorboard training logs for your method so that I can retrospect.

Thank you

**Comment by sreeharshaparuchur1** [comment]
The parameter ```NUM_UPDATES``` in ```/audio_separation/config/train/nearTarget.yaml``` is set to 67142. If the policy was updated 67142 times, that means it would've seen 67142 * 20 = 1342840 frames which is close to the number of frames my policy saw. If this parameter is indeed why my training concluded at said number of frames, please could you give some insight as to why the the value of ```NUM_UPDATES``` was set to be such.

**Comment by SAGNIKMJR** [comment]
1. Number of update steps (~150 mill) = number of updates (67142) * number of GPUs (8) * number of processes per GPU (14) * number of steps per update (20).

2. I don't think it's possible to reproduce the training with just 1 process and 1 GPU.
